code/modules/antagonists/nukeop/nukeop.dm

```diff
@@ -0,0 +1,659 @@
+/datum/antagonist/nukeop
+	name = ROLE_NUCLEAR_OPERATIVE
+	roundend_category = "syndicate operatives" //just in case
+	antagpanel_category = ANTAG_GROUP_SYNDICATE
```

Comment:
Idk what's going on here but you seem to have re-added the entirety of the nuke op datum, and I don't know why. I think someone moved its file recently so that might be your problem.